Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM mcr.microsoft.com/dotnet/sdk:7.0.404-1-bullseye-slim-amd64
+FROM mcr.microsoft.com/dotnet/sdk:8.0
+
+# "install" the dotnet 7 runtime so we can also run the NET 7 tests
+COPY --from=mcr.microsoft.com/dotnet/sdk:7.0 /usr/share/dotnet/shared /usr/share/dotnet/shared
```

Comment:
Should we not use the bullseye slim version anymore?

Reply:
same as above also it doesn't use the container it just copies over the files and I keep it the same as the FROM step

Reply:
It should be the same as above only for net 7